src/templates/rust-all/LICENSE.tftpl

```diff
@@ -0,0 +1,100 @@
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
```

Comment:
Who's MariaDB?

Reply:
They are the creators of the license: https://mariadb.com/
All the text outside of the "Parameters" box is to stay as it is.
Which is stated in the "Covenants of Licensor" part